dds/idl/dds_generator.h

```diff
@@ -510,47 +506,43 @@ inline void generateCaseBody(CommonFn commonFn,
       }
       rhs = "tmp.out()";
     } else if (br_cls & CL_ARRAY) {
-      forany = "      " + brType + "_forany fa = tmp;\n";
+      forany = "    " + brType + "_forany fa = tmp;\n";
       rhs = getWrapper("fa", br, WD_INPUT);
     } else {
       rhs = getWrapper("tmp", br, WD_INPUT);
     }
     be_global->impl_ <<
-      "    {\n"
```

Comment:
Was it intentional to remove the level of scope provided by curly braces here? If you have multiple cases like this, won't you now have two instances of "tmp" in the same scope and get those warnings about "jump to case label crosses initialization of XXX?"